docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -0,0 +1,281 @@
+specify a liveness probe, and specify a `restartPolicy` of Always or OnFailure.
+
+If you'd like to start sending traffic to a Pod only when a probe succeeds,
+specify a readiness probe. In this case, the readiness probe might be the same
+as the liveness probe, but the existence of the readiness probe in the spec means
+that the Pod will start without receiving any traffic and only start receiving
+traffic after the probe starts succeeding.
+
+If you want your Container to be able to take itself down for maintenance, you
+can specify a readiness probe that checks an endpoint specific to readiness that
+is different from the liveness probe.
+
+Note that if you just want to be able to drain requests when the Pod is deleted,
+you do not necessarily need a readiness probe; on deletion, the Pod automatically
+puts itself into an unready state regardless of whether the readiness probe exists.
+The Pod remains in the unready state while it waits for the Containers in the Pod
+to stop.
+
+## Pod and Container status
+
+For detailed information about Pod Container status, see
+[PodStatus](/docs/resources-reference/v1.5/#podstatus-v1)
+and
+[ContainerStatus](/docs/resources-reference/v1.5/#containerstatus-v1).
+Note that the information reported as Pod status depends on the current
+[ContainerState](/docs/resources-reference/v1.5/#containerstate-v1).
+
+## Restart policy
+
+A PodSpec has a `restartPolicy` field with possible values Always, OnFailure,
+and Never. The default value is Always.
+`restartPolicy` applies to all Containers in the Pod. `restartPolicy` only
+refers to restarts of the Containers by the kubelet on the same node. Failed
+Containers that are restarted by the kubelet are restarted with an exponential
+back-off delay. The delay is in multiples of sync-frequency (
+0, 1x, 2x, 4x, 8x ...) capped at five minutes, and is reset after ten minutes of
+successful execution. As discussed in the
+[Pods document](/docs/user-guide/pods/#durability-of-pods-or-lack-thereof),
+once bound to a node, a Pod will never be rebound to another node. This means
+that some kind of controller is necessary for a Pod to survive node failure,
+even if just a single Pod at a time is desired.
+
+Three types of controllers are available:
+
+- Use a [Job](/docs/user-guide/jobs/) for Pods that are expected to terminate,
+  for example, batch computations. Jobs are appropriate only for Pods with
+  `restartPolicy` equal to OnFailure or Never.
+
+- Use a [ReplicationController](/docs/user-guide/replication-controller/) or
+  [Deployment](/docs/user-guide/deployments/)
+  for Pods that are not expected to terminate, for example, web servers.
+  ReplicationControllers are appropriate only for Pods with a `restartPolicy` of
+  Always.
+
+- Use a [DaemonSet](/docs/admin/daemons/) for Pods that need to run one per
+  machine, because they provide a machine-specific system service.
+
+All three types of controllers contain a PodTemplate, which is also in a Pod. It
```

Comment:
nit: this sentence dosen't make sense. A PodTemplate is very similar to a pod, but the template isn't in the Pod (the templates metadata & spec are).